server/handlers/requestHandlers.go

```diff
@@ -45,5 +45,23 @@ func (h *Handlers) SaveNewUser(w http.ResponseWriter, r *http.Request) {
 }
 
 func (h *Handlers) LoginRequest(w http.ResponseWriter, r *http.Request) {
+	
```

Comment:
Can you add a log message like the new user at the beginning here? It could help in future debugging